frontend/src/host_orchestrator/docs/swagger.yaml

```diff
@@ -0,0 +1,134 @@
+basePath: /
+definitions:
+  v1.AndroidCIBuild:
+    properties:
+      branch:
+        description: The branch name. If omitted the passed `BuildID` will determine
+          the branch.
+        type: string
+      build_id:
+        description: |-
+          Uniquely identifies a branch's snapshot. If empty, the latest green snapshot of the used branch will
+          be used.
+        type: string
+      target:
+        description: A string to determine the specific product and flavor from the
+          set of builds.
```

Comment:
nit: not "from the set of builds". Maybe just "determine the specific device product and flavor"?

Reply:
Got it, this sentence is autogenerated from the existing documentation at https://github.com/google/android-cuttlefish/blob/1b4c058404a4e7c638b6bd08f0d1b3f51f829919/frontend/src/host_orchestrator/api/v1/messages.go#L71 I'll modify the comment with your suggestion in a follow up PR.

Reply:
Fix is sent for review in https://github.com/google/android-cuttlefish/pull/1262/commits/8540000a5b5eecb2d68d1cf58a44c7013f38123d